Src/FluentAssertions/FluentAssertions.csproj

```diff
@@ -91,11 +91,11 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'net47'">
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.0" />
-    <PackageReference Include="System.Net.Http" Version="4.3.4" />
     <Reference Include="System.Configuration" />
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
     <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Net.Http" />
```

Comment:
FWIW, I found in testing that this change wasn't enough for the nuspec to express this as a framework dependency. That's why in my PR I add an additional item.

Reply:
Let's continue with your PR